### DISCUSSION TITLE
how to use nvidia A 100 80 vgpu using singularity

### DISCUSSION THREAD

**Discussion by Ajazul** [sylabs/singularity#2471] [Q&A]

**Comment by dtrudg** [comment]
Hi @Ajazul - please provide some more information, such as:

* What version of Singularity are you using?
* How has the GPU been partitioned into vGPUs?
* What container / software you are trying to run.

I have moved this issue to a discussion, as it is a usage question and not a bug report or feature request.

**Reply by Ajazul** [reply]
singularity-ce version 3.11.5-1.el8
mig instances 
podman

**Reply by dtrudg** [reply]
I'm a little confused here - are you trying to run podman inside a Singularity container? Can you provide examples of how you try to start the Singularity container, and the commands you are running? What errors / behaviour do you see?

It's unlikely anyone will be able to offer any help without you providing a lot more detail.

If you are trying to run podman inside of Singularity with vGPU support, I'm afraid this isn't something I can offer assistance with. Nesting different container runtimes with GPU support is quite unlikely to work. SingularityCE's approach to binding NVIDIA libraries will potentially result in them being exposed in different places than podman's GPU support expects.

Instead, I suggest looking into the OCI-mode in Singularity 4.0, which allows running container more similarly to other OCI runtimes like podman, without nesting.

**Comment by dtrudg** [comment]
Closing as no follow-up information has been provided. Feel free to re-open it if you can give more information that would help to answer the question.
